### DISCUSSION TITLE
Need a public-facing name for the DAISY Pipeline; we need a name!

### DISCUSSION THREAD

**Discussion by GeorgeKerscher** [daisy/pipeline#775] [Ideas]
Richard and George are doing our first interview for Eyes on Success podcast on February 23. We realize that DAISY Pipeline is not the ideal name for a public-facing product.  We did some brainstorming, but clearly there is not enough time before the interview to make a decision. However, before our launch we need to come up with a name. For the interview, we will give the url daisy.org/converter with a few versions like convert and conversion, which can redirect to whatever we decide on in the future.

So, let us all start thinking about a proper name.

**Comment by ways2read** [comment]
In discussions with Microsoft and partner organizations we have talked about the "multi format converter" or "easy to use converter tool".

EasyConverter is already taken (by Dolphin). Otherwise, that would have worked well, emphasizing the simplicity and the core functionality.

Converter Central references a place that brings together various elements. It's a bit long though.

DAISY 123, DAISY 1-2-3, DAISY 1 2 3, or DAISY 1 to 3 reference easy steps.

In any event, "Powered by DAISY Pipeline" seems like an appropriate strapline and references the underlying 'engine'.

**Comment by GeorgeKerscher** [comment]
Would the word Simple or Symply work in the name? Simply Convert or Simple Converter?

**Comment by SarahMorleyWilkins** [comment]
I’ve obviously not been part of the discussions about this to date, but has ‘DAISY Converter’ been discounted already? That doesn’t need any explanation, and could have a sub-title which expands it, something like:

DAISY Converter: making it easy to convert files into accessible formats.

I like the ‘Powered by DAISY Pipeline’ strapline.
Sarah

**Comment by GeorgeKerscher** [comment]
Hello,

In our interview with Eyes on Success, we gave the url daisy.org/converter

**Comment by avneeshsingh** [comment]
Renaming or naming such an interdependent product in a rush is never a good idea. We need to evaluate and consider all the aspects before making a decision. We should let the user interface evolve more, before thinking about giving it a creative name.

In fact we have started discussing the name of DAISY Pipeline user interface here but we did not test the feasibility of this action in Microsoft DAISY team meetings or in management team meetings. 

DAISY Pipeline is a well established product with well established brand image for last 17 years.
The user interface is not a new product, it is interwoven with DAISY Pipeline. Even the names of conversion scripts shown in the user interface are fetched from DAISY Pipeline web service. It clearly has DAISY Pipeline signature in conversions and workflow. The user interface product cycle, development workflow  and distribution is also embedded with existing DAISY Pipeline. This is the most efficient way of driving such a huge cross platform Pipeline project with such a small team.

It is important for us to maintain the existing brand of DAISY Pipeline. If we want to think about updating name of just the user interface, then it goes quite deep into product cycle. We need to somewhat disassociate the user interface from existing Pipeline, we would need to maintain the user interface as a separate product, provide separate cycle of releases. 
It raises a red flat, being such a small team should we take over such an overhead.
We intentionally did not mention name DAISY Pipeline in Microsoft project because we were not clear about their funding priorities. So, it was done to be on safe side, and not for renaming it.

As per the current product management (development, distribution, maintenance etc.) we have existing Pipeline web service, which has the name DAISY Pipeline 2 and we see the user interface as the “DAISY Pipeline App” or “DAISY Pipeline 2 App”. (I would like to avoid use DAISY Pipeline "2" App at this stage because we are discussing integrating Pipeline 2 and Pipeline1 in development team.

So, I believe that we should not rush for naming, and wait for next development cycle of the user interface. Also, the development team should have full focus on delivering results in this last month instead of doing strategic and technical analysis of changing the name.

We have two main pieces in DAISY Pipeline package, DAISY Pipeline 2 and DAISY Pipeline App. In short term, what we really need is to explain to new audience, what is “DAISY Pipeline App”. 
The best way to do it at this time is to have an explainer tag line or strap line. (the name Mc Donald’s does not make it a weaker burger brand than Burger King)
An example is:
DAISY Pipeline App
Simplifying your accessible formats conversions.
We can work on the tag/strap line for making it attractive and self-explanatory.

**Comment by ways2read** [comment]
The new app is a desktop version of the DAISY Pipeline. Perhaps in the future it will evolve into something else, but right now that is what it is. The beta versions will reach the first of the target organizations in the next day or so, so it will be interesting to get their feedback.

I agree, a new name for the launch is not viable (and probably not desirable). This is mostly about how we refer to what we have and the messaging emphasizing easy to install and use.

I also agree with not referring to "Pipeline 2", so something along the lines of "DAISY Pipeline app" (this specifically refers to the new desktop tool) followed by:
- easy-to-use multi-converter tool
- simple converter tool
- multi-format converter
- accessible book converter

We have some work to do on the website content and documentation to tidy up the existing content, and some decisions about where we place information about the new tool. That includes what we do about the section on Pipeline 1, and the references to the desktop app that are currently on the site.

I don't think a discussion about web pages is best done in GitHub. How shall we do this?

**Comment by bertfrees** [comment]
> I don't think a discussion about web pages is best done in GitHub. How shall we do this?

Are you in our Slack community?

**Comment by avneeshsingh** [comment]
Slack is not so efficient for screen reader users. Reading it is fine, but responding to it while reading is not straight forward.

**Comment by ways2read** [comment]
I don’t want to divert our developers from their important development tasks. At the same time, I don’t want to make decisions about these web pages without inputs from at least someone from that area.

Our approach is that user information is on the DAISY website and developer information is on GitHub. Possibly we could slim down the information on the DAISY site about Pipeline 1 and Pipeline 2, point the developers more to the relevant places off the DAISY site, and by doing so make it less confusing for the Pipeline app end users to find the information they need.

**Comment by avneeshsingh** [comment]
The Pipeline webpages on daisy.org provides formal entry to the users in respective projects. I think we can simplify them a little.

Specifically for Pipeline 2, we should introduce the project and provide different interfaces it provides, CLI, webservice etc.

And on the same page we should have a section for breaking news where we introduce the DAISY Pipeline app. And this breaking news section should have link to user oriented documentation.

Regarding developer oriented information, I think that we can point people to GitHub and have a GitHub.io page there for technical users of Pipeline.

**Comment by bertfrees** [comment]
I think it's strange if the name of an application ends with "app". It's like if the name of a car or a restaurant ends with "car" or "restaurant".

**Comment by avneeshsingh** [comment]
We need to differentiate user interface from CLI.

From usability side it is very important for non-visual users to know difference between CLI and Application. The screen reader users know this by reading the title of the active window. If CLI title and Application title say the same thing, it makes it very confusing.

 

In Ace also we used Ace for CLI and Ace App for user interface.

**Comment by bertfrees** [comment]
You could use the term "desktop" instead of "application" or "app".

**Comment by avneeshsingh** [comment]
I am OK with any of Desktop or App. We just need a user friendly differentiation.

**Comment by GeorgeKerscher** [comment]
I normally refer to software on my computer as a desktop application. When I hear App, I think of a mobile phone or tablet. Just my view of things.

**Comment by ways2read** [comment]
The term app has widened to include desktop applications, so I don't find it strange to refer to the Ace app, for example. If we didn't need to make the distinction, then we wouldn't need to include app. It's just Obi, for example.

But we do, and I think Pipeline app works well.

Richard
________________________________

**Comment by marisademeglio** [comment]
Or, we could rename the CLI to "DAISY Pipeline command line" and leave the app "DAISY Pipeline". I know GUI development usually comes later so that's why our naming has evolved the way it has, but we should be a step ahead of that in our product naming.

**Comment by bertfrees** [comment]
I agree with Marisa. Even if we have other user interfaces that are called DAISY Pipeline, there is nothing that stops us from rename those instead. Besides, the idea is that the new interface will replace both other graphical user interfaces we had: the older desktop application, and eventually also the web-based application.

For me this new application is just the newest incarnation of Pipeline. It's the same as it was before but with a new outfit.

Also for the distribution side of things I see no problem. The new application will not only replace the existing graphical user interfaces, it will be the new main way to distribute the web server (already included in the new application today because we use it to communicate between the front- and backend, and referred to as "the engine" in the user interface).

The only other thing that is left is the command line client. AFAIK we already call it "command line tool" or "DAISY Pipeline 2 CLI" on our distribution channels.

To what Avneesh said about screen reader users: if you were running the CLI, wouldn't the title of the active window be something like "dp2" (the name or path of CLI executable), or "Terminal"? There's not a conflict I think. I can see more of a conflict with DAISY Pipeline 1 for screen reader users.

**Comment by GeorgeKerscher** [comment]
When I go to sites and their downloads are not clear, I don't know which one to get. I thinking having a CLI and an App makes that clear. Both get the same conversion functionality and scripts, I believe.

Best
George

**Comment by GeorgeKerscher** [comment]
When I go to a site and there are options for the download, I like it when it is clear what I should be getting. For this reason, having one as the CLI and the App seems right.

**Comment by bertfrees** [comment]
Yes, we should make our downloads clear. No question.

From our websites there has always been one package per platform. The downloads page explains what exactly the packages contain (sometimes with the web server and command line tool included, sometimes without). The command line tool by itself (clearly called CLI), as well as other more specialized packages, are only available via Github.

With the new installer replacing a bunch of other packages, things will become simpler. We can still offer only one package per platform, as we did before, but from now on it will actually include everything (also the command line interface) and no other packages will be available. So no confusion possible. That's the idea.

I don't think we need to call the new application "DAISY Pipeline App". We can be extra careful we make the download links clear, but the application itself (in the start menu, in the title bar, in the dock, in the tray, in the menus of the application itself, ...) can be called "DAISY Pipeline".

**Comment by ways2read** [comment]
Pipeline is the technology we offer for file conversion/validation. And then within it we refer to specific elements: the scripts, the command line interface, and the app.

When we download and install, we're installing DAISY Pipeline. When we're running the new GUI, that's the Pipeline app.

Richard
________________________________

**Comment by bertfrees** [comment]
@ways2read OK yes, makes sense. We're running a specific component that is part of the whole software suite (which is installed via a single installer). You're right that if we reserve the name "DAISY Pipeline" for the whole suite, it may be a good idea to not use the name to also refer to a part of it. I'm not 100% convinced that it's absolutely necessary in all cases though.

But anyway, all this doesn't change the fact that I don't like the "app" in "DAISY Pipeline app" 😅.

**Comment by marisademeglio** [comment]
There are two issues converging in this thread: 

- product name
- accessible window title for optimal screen reader usage (which started over in another issue daisy/pipeline-ui#70)

We are immediately addressing daisy/pipeline-ui#70 this week and then we can review this issue (#77) again in light of those changes.

**Comment by bertfrees** [comment]
@marisademeglio I'm fine with closing the issue. I think "DAISY Pipeline" is a perfect name. But I don't know if everyone agrees.

**Comment by prashantverma2014** [comment]
The very first DAISY Pipeline is still in use  at many places. The DAISY Pipeline 2 is live on our website.

Hence this new app, which is not exactly an update to the previous ones, should have different name or numbering.

DAISY Pipeline 3 is OK. Or DAISY Pipeline app can also be considered.

**Comment by avneeshsingh** [comment]
Looking beyond the existing names and versions, we should think from perspective of development plans.

The plan is to have single installer that contains command line, USER interface etc. One package that installs everything. So, having different user-oriented names of different pieces of package will confuse the users further.

Furthermore, there are plans to integrate Pipeline 1 with Pipeline 2 and operate Pipeline 1 through latest version of Pipeline. It is possible that this integration takes a little different shape depending on technical details.

So, we should try to keep uniform names for different pieces of DAISY Pipeline packaged together.

I think it is too early to visit naming of DAISY Pipeline. Let the plans move forward, if things shape up differently then we should visit renaming of Pipeline.
